components/brave_shields/resources/panel/components/advanced-controls-content/index.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react'
+
+import * as S from './style'
+import { CaratStrongDownIcon } from 'brave-ui/components/icons'
+import Toggle from '../../../../../web-components/toggle'
+import { getLocale } from '../../../../../common/locale'
+
+function GlobalSettings () {
+  return (
+    <S.SettingsBox>
+      <S.SettingsTitle>{getLocale('braveShieldsGlobalSettingsTitle')}</S.SettingsTitle>
+      <a href='brave://settings/shields' target='_blank'>
```

Comment:
This doesn't work. In webui, we should use `chrome` scheme.
and anchor tag doesn't work in panel.